srtcore/core.h

```diff
@@ -1139,8 +1184,30 @@ class CUDT
 
         int64_t sndDuration;                // real time for sending
         time_point sndDurationCounter;         // timers to record the sending Duration
+
+        AverageMetricUsage<duration> tdAverageResponseTime;
+        MaxMetricUsage<duration> tdMaxResponseTime;
     } m_stats;
 
+    /// This function records the passed current time as the last response time.
+    /// Before doing it, however, it checks if there exist any previous such time
+    /// and updates statistics accordingly.
+    void calculateResponseTime(const time_point& now)
```

Comment:
If it records the time, then a better name could be `recordResponseTime`.
Otherwise this operation is unexpected: `m_tsLastRspTime = now;`.